ride-share

```diff
@@ -0,0 +1,147 @@
+driver_data = [ { "DR0001"=> { "ride1"=> {"date"=> "3rd Feb 2016",
+                                          "cost"=> 10,
+                                          "riderid"=> "RD0003",
+                                          "rating"=> 3},
+                               "ride2"=> {"date"=> "3th Feb 2016",
+                                          "cost"=> 30,
+                                          "riderid"=> "RD0015",
+                                          "rating"=> 4},
+                               "ride3"=> {"date"=> "5th Feb 2016",
+                                          "cost"=> 45,
+                                          "riderid"=> "RD0003",
+                                          "rating"=> 2}},
+                  "DR0002"=> { "ride1"=> {"date"=> "3rd Feb 2016",
+                                          "cost"=> 25,
+                                          "riderid"=> "RD0073", 
+                                          "rating"=> 5},
+                               "ride2"=>{"date"=> "4th Feb 2016",
+                                         "cost"=> 15,
+                                         "riderid"=> "RD0013",
+                                         "rating"=> 1},
+                               "ride3"=> {"date"=> "5th Feb 2016",
+                                          "cost"=>35,
+                                          "riderid"=> "RD0066",
+                                          "rating"=> 3}},
+                  "DR0003"=> { "ride1"=>{"date"=> "4th Feb 2016",
+                                         "cost"=> 5,
+                                         "riderid"=> "RD0066",
+                                         "rating"=> 5},
+                               "ride2"=> {"date"=> "5th Feb 2016",
+                                          "cost"=>50,
+                                          "riderid"=> "RD0003",
+                                          "rating"=> 2}},
+                  "DR0004"=> { "ride1"=> {"date"=> "3rd Feb 2016",
+                                          "cost"=> 5,
+                                          "riderid"=> "RD0022",
+                                          "rating"=> 5},
+                               "ride2"=>{"date"=> "4th Feb 2016",
+                                         "cost"=> 10,
+                                         "riderid"=> "RD0022",
+                                         "rating"=> 4},
+                               "ride3"=> {"date"=> "5th Feb 2016",
+                                          "cost"=>20,
+                                          "riderid"=> "RD0073",
+                                          "rating"=> 5}}}]
+
+
+
+#the number of rides for each driver
+def number_of_rides (arr) 
+number_of_rides = []
+ arr.each do |drivertrip|
+number_of_rides = drivertrip.map { |k, v| [k, v.count] }.to_h 
+
+
+end
+return number_of_rides
+end
+
+number_of_rides = number_of_rides(driver_data)
+puts number_of_rides
```

Comment:
You should really tell the user that this data has to do with the number of rides for each driver.

Consider looping through `number_of_rides` and printing each driver neatly on a separate line.